lib/components/artist/artworks/index.js

```diff
@@ -113,26 +107,7 @@ export default Relay.createContainer(Artworks, {
           artworks
           for_sale_artworks
         }
-        for_sale_artworks: artworks_connection(sort: partner_updated_at_desc, filter: IS_NOT_FOR_SALE, first: $pageSize) {
-          edges {
-            node {
-              image {
-                id
-              }
-              ${Artwork.getFragment('artwork')}
-            }
-          }
-        }
-        not_for_sale_artworks: artworks_connection(sort: partner_updated_at_desc, filter: IS_NOT_FOR_SALE, first: $pageSize) {
-          edges {
-            node {
-              image {
-                id
-              }
-              ${Artwork.getFragment('artwork')}
-            }
-          }
-        }
+        ${ArtworksGrid.getFragment('artist')}
```

Comment:
You probably need to pass in the filter variable here, right?

And I assume the fact that there’s only 1 grid now is just because you’re trying to get 1 to work first?